### DISCUSSION TITLE
리트코드 스터디 3주차 모임

### DISCUSSION THREAD

**Discussion by SamTheKorean** [DaleStudy/leetcode-study#382] [공지사항]
스터디 2기 참여자 여러분, 안녕하세요. Sam입니다👋

세 번째 모임도 전 모임과 같이 지원자를 받고 각각 문제에 대해서 설명 및 논의하는 시간을 가지겠습니다!

문제당 한 분씩 풀이를 할 예정이고, 신청은 선착순으로 각 주차에 있는 아직 풀이 희망자가 없는 문제 중 원하시는 문제와 디스코드 명을 one-bit-integer / Sam 과 같은 형식으로 댓글로 남겨주시면 됩니다!ㅎㅎ

다음 모임도 이전과 마찬가지로 아래 링크로 RSVP 부탁드리겠습니다! (https://discord.gg/5N4BSC6M?event=1277446411537092653)

**Comment by TonyKim9401** [comment]
combination-sum / Tony

**Reply by TonyKim9401** [reply]
발표용 PPT 미리 공유해둡니다! 편하게 확인해주세요!
https://docs.google.com/presentation/d/1QCNLi94bXvjLo4jP55y8sSehUF8jp7-2bUQQFn9_6ho/edit?usp=sharing

**Comment by jdalma** [comment]
Coin Change / 정현준

**Comment by EgonD3V** [comment]
Climbing stairs / EGON
